app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    copy_command, original, copy = command.split()
+    if original != copy:
```

Comment:
u should check if cp command is == "cp" 
![image](https://user-images.githubusercontent.com/103201010/220620594-5df0b36b-1368-4d57-a3b0-aedc7587d79b.png)